### DISCUSSION TITLE
How to build on Windows with Visual Studio?

### DISCUSSION THREAD

**Discussion by DMS-CPP** [WebAssembly/binaryen#3902] [Q&A]
I generated the Visual Studio solution using CMake, and then I tried to build the project, but I get this error:
```
1>------ Build started: Project: passes, Configuration: Debug x64 ------
1>Generating WasmIntrinsics.cpp
1>The system cannot find the path specified.
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Microsoft\VC\v160\Microsoft.CppCommon.targets(234,5): error MSB6006: "cmd.exe" exited with code 3.
1>Done building project "passes.vcxproj" -- FAILED.
```

**Comment by marad** [comment]
I've built binaryen on Windows yesterday just fine. One thing important to note is that it worked for me only in x64 Native Tools Command prompt, which is basically a terminal with all the paths properly set up for compilation.
